modules/admin/ref-rbac-rest-api-endpoints.adoc

```diff
@@ -1,7 +1,209 @@
 [id='ref-rbac-rest-api-endpoints_{context}']
 = Supported RBAC REST API endpoints
 
-The RBAC REST API provides the following endpoints for managing permission policies and roles in the {product-short} and for retrieving information about the policies and roles.
+The RBAC REST API provides the following endpoints for managing roles, permissions, and conditional policies in the {product-short} and for retrieving information about the roles and policies.
+
+== Roles
+
+The RBAC REST API supports the following endpoints for managing roles in the {product}.
+
+[GET] /api/permission/roles::
++
+--
+Returns all roles in {product-short}.
+
+.Example response (JSON)
```

Comment:
Instead of having each of these say the same thing, consider giving them specific titles so the user always knows what they are looking at. Like _Example JSON output for the [endpoint being exemplified]_

Reply:
Thanks for the feedback. This structure represents the API documentation aspects, detailing all components consistently, even when accessing endpoints through Swagger, which all users use to access the API endpoints.